app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -214,7 +214,7 @@ type CustomizableFileValue @doc(description: "CustomizableFileValue defines the
 }
 
 interface MediaGalleryInterface @doc(description: "Contains basic information about a product image or video.") @typeResolver(class: "Magento\\CatalogGraphQl\\Model\\MediaGalleryTypeResolver") {
-    url: String @doc(description: "The URL of the product image or video.") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\MediaGallery\\Url")
+    url(width: Int, height: Int): String @doc(description: "The URL of the product image or video.") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\MediaGallery\\Url")
```

Comment:
That might work but I'm not sure it's consistent enough to have these parameters for the URL field. It might be not obvious in the end. 

@paliarush could you share your thoughts on this approach, please? Thanks a lot!